src/main/java/jenkins/plugins/openstack/compute/slaveopts/BootSource.java

```diff
@@ -272,21 +272,23 @@ public int hashCode() {
             @InjectOsAuth
             public ListBoxModel doFillNameItems(@QueryParameter String name,
                                                 @QueryParameter String endPointUrl,
+                                                @QueryParameter boolean ignoreSsl,
                                                 @QueryParameter String credentialId,
                                                 @QueryParameter String zone) {
-                return makeListBoxModelOfAllNames(name, endPointUrl,credentialId, zone);
+                return makeListBoxModelOfAllNames(name, endPointUrl, ignoreSsl, credentialId, zone);
             }
 
             @Restricted(DoNotUse.class)
             public FormValidation doCheckName(@QueryParameter String value,
                     // authentication fields can be in two places relative to us.
                                               @RelativePath("../..") @QueryParameter("endPointUrl") String endPointUrlCloud,
                                               @RelativePath("../../..") @QueryParameter("endPointUrl") String endPointUrlTemplate,
+                                              @QueryParameter boolean ignoreSsl,
```

Comment:
I am afraid we need this to be duplicated as all the other cloud fields in the signature for it to work.